### ISSUE TITLE
upstream wishlist item: public or objective verificiationMethod specification?

### ISSUE THREAD

**Issue by bumblefudge** [w3c-ccg/did-pkh#18]
on today's call, we discussed how useful it would be if the spec pointed to canonical references explaining the mechanics that would facilitate contributions from the community and new implementations.  Even an upstream PR to explain BlockchainVerification2021 here might help:
https://or13.github.io/lds-blockchain2021/#blockchainAccountId

**Comment by bumblefudge** [comment]
Cf today's thread on the CCG list for context:
https://lists.w3.org/Archives/Public/public-credentials/2021Dec/0014.html

**Comment by bumblefudge** [comment]
Also note: the #blockchainAccountId and #blockchainVerification2021 both point to a w3id.org alias that redirects here, which is not very detailed:
https://or13.github.io/lds-blockchain2021/#blockchainAccountId

Perhaps at a later date, the group could propose an upstream PR and/or add some more explanatory text, whether in the spec or as a reference.

**Comment by bumblefudge** [comment]
See also [this thread](https://github.com/w3c-ccg/vc-api/issues/67#issuecomment-966861272) over at vc-api repo

**Comment by bumblefudge** [comment]
Found [this thread on the DID-WG's public list](https://lists.w3.org/Archives/Public/public-did-wg/2022Jan/0005.html) explaining how straightforward the definition (in LD terms) of a vM is-- basically just a definition by example, a `type` registration, and a URL for context definition.  Should we consider writing one in this group?

**Comment by clehner** [comment]
Discussing on call...
BlockchainVerification2021 makes sense I think; also there is https://identity.foundation/EcdsaSecp256k1RecoverySignature2020/ which is specific to Secp256k1 but works well for verification against public key hashes because of using EC Recovery.

**Comment by clehner** [comment]
Mentioned on today's call (#28). @bumblefudge maybe we need your input here?

It may be relevant to note that BlockchainVerificationMethod2021 is not mentioned in https://w3c.github.io/vc-wg-charter/ - but [EcdsaSecp256k1RecoverySignature2020](https://github.com/decentralized-identity/EcdsaSecp256k1RecoverySignature2020) is being proposed to mention as a possible input document for rechartered VCWG (https://github.com/w3c/vc-wg-charter/pull/105).

**Comment by bumblefudge** [comment]
Yes, glad this is in scope for VCWG; if they are specifying it (including the variant with the extra bit) then we could just leave this open as a tracking issue.
